Purchases/Purchasing/Purchases.swift

```diff
@@ -1401,7 +1401,11 @@ public extension Purchases {
      * - returns ``RefundRequestStatus``: The status of the refund request.
      * Keep in mind the status could be ``RefundRequestStatus/userCancelled``
      *
-     * If the request was unsuccessful or no active entitlements could be found for the user, an `Error` will be thrown.
+     *- throws: If the request was unsuccessful, no active entitlements could be found for the user,
+     * or multiple active entitlements were found for the user, an `Error` will be thrown.
+     *
+     *- warning: This method should only be used if your user can only have a single active entitlement at a given time.
```

Comment:
looking at this again, I think it should have been a `- important:` so it does the yellow bubble thing and not red

https://developer.apple.com/documentation/xcode/formatting-your-documentation-content

Apple says to use `important` for "Important information, such as a requirement.", which seems to fit the bill here.

Sorry for suggesting the wrong one in the first place!